zingolib/CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## 1.3.2 2023-10
```

Comment:
What's this number?

Reply:
I think your changes are going into `Unreleased` so we can just delete this number, which I copied form the keepachangelog URL.